library/src/main/java/com/novoda/downloadmanager/lib/DownloadsRepository.java

```diff
@@ -195,6 +198,71 @@ public void setDownloadSubmitted(FileDownloadInfo info) {
         contentResolver.update(info.getAllDownloadsUri(), contentValues, null, null);
     }
 
+    @Nullable
+    String getCurrentDownloadingBatchId() {
+        return getCurrentBatchIdWithColumnStatus(DownloadStatus.RUNNING);
```

Comment:
does this assume there's only 1 download downloading at a time?? what if there are more?